src/Autofac/Autofac.csproj

```diff
@@ -4,6 +4,7 @@
     <Description>Autofac is an IoC container for Microsoft .NET. It manages the dependencies between classes so that applications stay easy to change as they grow in size and complexity.</Description>
     <VersionPrefix>4.9.3</VersionPrefix>
     <TargetFrameworks>netstandard2.0;netstandard1.1;net45</TargetFrameworks>
+    <LangVersion>latest</LangVersion>
```

Comment:
[This is neat](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/configure-language-version) and something I didn't know about. 👍